Client/clientSocket.py

```diff
@@ -24,13 +25,13 @@
 disable_multiplayer = Config.get_config().Disable_Multiplayer
 
 
-async def start_connections(server_config: ServerConfig, set_up_dto: SetUpDto, clientOutput: QListWidget):
+async def start_connections(server_config: ServerConfig, set_up_dto: Optional[SetUpDto], clientOutput: Optional[QListWidget]) -> None:
```

Comment:
Although the `SetUpDto` is `Optional`, I'd prefer it to be as it was originally. It's a very rare edge case where it isn't `Optional`, that I'll likely be removing soon.